projects/plugins/jetpack/modules/widgets/top-posts.php

```diff
@@ -681,8 +765,8 @@ public function get_posts( $post_ids, $count, $types = array( 'post', 'page' ) )
 			$posts[] = compact( 'title', 'permalink', 'post_id', 'post_type' );
 			$counter++;
 
-			if ( $counter == $count ) {
-				break; // only need to load and show x number of likes
+			if ( $counter == $count ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
```

Comment:
I wonder if we should just cast `$count` to an integer at the top of the function and make this strict.

Although I see the filter just below for some reason claims `$count` will be a string. Weird.

Reply:
This was one of those cases where I prefer to leave it as it is to avoid breaking something that doesn't immediately seem obvious to me. I think I'll leave it like that for now. :)